google-cloud-core/src/main/java/com/google/cloud/Policy.java

```diff
@@ -51,11 +47,26 @@
 public final class Policy implements Serializable {
 
   private static final long serialVersionUID = -3348914530232544290L;
-
-  private final Map<Role, Set<Identity>> bindings;
+  private final List<Binding> bindingsList;
   private final String etag;
   private final int version;
 
+  /*
+   * Return false if Policy is version 3 OR bindings has a conditional binding.
+   * Return true if Policy is version 1 AND bindings does not have a conditional binding.
+   */
+  private static boolean checkVersion(int version, List<Binding> bindingsList) {
```

Comment:
I might call this `isVersion`. The `check*` methods in guava throw exceptions when conditions are not met, whereas this method is returning a boolean